examples/CustomCollectionView/Sample/MosaicCollectionLayoutDelegate.m

```diff
@@ -36,11 +36,13 @@ - (instancetype)initWithNumberOfColumns:(NSInteger)numberOfColumns headerHeight:
 
 - (ASScrollDirection)scrollableDirections
 {
+  ASDisplayNodeAssertMainThread();
```

Comment:
I intentionally added to tell whoever reading this example that these methods are called on main. Some rarely read the documentation :) I can definitely remove it if you think it's excessive.